.ruby-version

```diff
@@ -0,0 +1 @@
+0.2.0
```

Comment:
mm, this should be the ruby version (same we have in travis) so 2.4.2 right?

Reply:
I don't think so. The current version of the client library is `0.1.1` (https://github.com/openregister/registers-ruby-client/releases).

Reply:
The file `.ruby-version` states the version of Ruby you are expecting to use, not the version of the gem (library) itself. It is a convention that different ruby version tools agreed on some time ago (e.g. https://rvm.io/workflow/projects)